site/app/config/services.yml

```diff
@@ -4,8 +4,12 @@ parameters:
 #    parameter_name: value
 
 services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
     # Twig date extension - http://twig-extensions.readthedocs.io/en/latest/date.html
-    twig.extension.date:
-       class: Twig_Extensions_Extension_Date
-       tags:
-            - { name: twig.extension }
+    twig.extension.date: # html template
+        class: Twig_Extensions_Extension_Date # getagged class
+
```

Comment:
Your changes broke the extension: http://www.librecores.devel/project/list is showing " Updated diff.ago.month " only. "diff.ago.month" is the translation string, i.e. we're missing the translations. This behavior is caused by autowire injecting a TranslatorInterface into the constructor of the extension. If that happens, the extension thinks it can be translated -- but we don't have any translations available. Adding `autowire: false` solves this problem.

Additionally, since version 1.5.0 of the twig extensions namespaced classes were added, so we can now make use of the classname=servicename paradigm that you're using below.

```patch
diff --git a/site/app/config/services.yml b/site/app/config/services.yml
index 0c3d3e0..c667729 100644
--- a/site/app/config/services.yml
+++ b/site/app/config/services.yml
@@ -10,6 +10,9 @@ services:
         public: false
 
     # Twig date extension - http://twig-extensions.readthedocs.io/en/latest/date.html
-    twig.extension.date: # html template
-        class: Twig_Extensions_Extension_Date # getagged class
-
+    Twig\Extensions\DateExtension:
+        # XXX: autowiring is disabled to make sure no TranslatorInterface is
+        # passed to the constructor. If it is, we need to add translations
+        # for date.ago.xxx keys as discussed here:
+        # http://twig-extensions.readthedocs.io/en/latest/date.html#translation
+        autowire: false
diff --git a/site/composer.json b/site/composer.json
index adb0e92..21a706a 100644
--- a/site/composer.json
+++ b/site/composer.json
@@ -21,7 +21,7 @@
                "doctrine/dbal" : "~2.6.0",
                "doctrine/doctrine-bundle" : "~1.7.0",
                "doctrine/doctrine-fixtures-bundle" : "<2.5",
-               "twig/extensions" : "~1.0",
+               "twig/extensions" : ">=1.5.0",
                "sensio/distribution-bundle" : "~5.0.18",
                "sensio/framework-extra-bundle" : "~4.0.0",
                "incenteev/composer-parameter-handler" : "~2.0",
```

(You need to run `cd /var/www/lc/site; ./bin/console cache:clear` inside the VM to see the changes.)